draft-sheffer-wimse-s2s-protocol.md

```diff
@@ -114,7 +114,7 @@ policy management and message authorization are out of scope of this document.
 The high-level message flow is as follows:
 
 * Workload A obtains a credential from the Identity Server. This happens periodically, e.g. once every 24 hours.
-* Workload A makes a REST call into Workload B. This is a regular REST call, with the additional protection
+* Workload A makes an HTTP call into Workload B. This is a regular HTTP call, with the additional protection
```

Comment:
I don't think I've ever heard the term "HTTP call". Can we think of a better term?

Reply:
I feel like I hear the term used all the time but that doesn't necessarily it's widely used or understood. How about just good ol' "HTTP request"? 

```suggestion
* Workload A makes an HTTP call into Workload B. This is a regular HTTP request, with the additional protection
```